src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/components/options_list_popover_invalid_selections.tsx

```diff
@@ -97,7 +102,7 @@ export const OptionsListPopoverInvalidSelections = () => {
           invalidSelections.size
         )}
         options={selectableOptions}
-        listProps={{ onFocusBadge: false, isVirtualized: false }}
+        listProps={{ onFocusBadge: false }}
```

Comment:
By enable virtualization in the `EuiSelectable` component in the invalid selections component, we get a list with a fixed height with scrolling enabled similar to the list of available options above it. This fixes the issue described in https://github.com/elastic/kibana/issues/222121.